src/main/java/entities/Task.java

```diff
@@ -1,12 +1,12 @@
-
 package entities;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 
 /**
  * An entity class representing a Task
  */
-public class Task {
+public class Task implements Serializable {

```

Comment:
This should change -- we only need User to be serializable as we are only storing User.